pkg/agent/datapath/config.go

```diff
@@ -81,15 +79,19 @@ func getDatapathConfig() (*datapath.Config, error) {
 	return dpConfig, nil
 }
 
-func setAgentConf(datapathManager *datapath.DpManager, k8sReader client.Reader) {
+func SetAgentConf(datapathManager *DpManager, k8sReader client.Reader) {
 	k8sClient := k8sReader.(client.Client)
 	agentInfo := datapathManager.AgentInfo
 	agentInfo.EnableCNI = true
-	agentInfo.NodeName = os.Getenv(constants.AgentNodeNameENV)
+
+	nodeName, _ := os.Hostname()
+	nodeName = strings.ToLower(nodeName)
+	agentInfo.NodeName = nodeName
 
 	node := corev1.Node{}
+
 	if err := k8sClient.Get(context.Background(), client.ObjectKey{
-		Name: agentInfo.NodeName,
+		Name: nodeName,
```

Comment:
Why change this part?

Reply:
this is a misoperation and has been modified back